com.corifeus.onenote.yml

```diff
@@ -0,0 +1,71 @@
+app-id: com.corifeus.onenote
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: stable
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node12
+modules:
+  - name: zypak
```

Comment:
Zypak is part of the base app.

Reply:
It isn't :/
![Screenshot from 2020-09-05 23-12-08](https://user-images.githubusercontent.com/31540351/92313552-8a376e00-efbc-11ea-8ebc-020f4eb2b003.png)

Reply:
It absolutely is: https://github.com/flathub/org.electronjs.Electron2.BaseApp/blob/branch/19.08/org.electronjs.Electron2.BaseApp.yml#L81